kie-spring-boot/kie-spring-boot-samples/kie-server-spring-boot-sample/pom.xml

```diff
@@ -133,6 +143,13 @@
             </execution>
           </executions>
         </plugin>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-surefire-plugin</artifactId>
+          <configuration>
+            <reuseForks>false</reuseForks>
```

Comment:
reuseForks false executes each test class in its own JVM process (if not, clashes with other tests). According to the documentation, _it creates the highest level of separation for the test execution_

Reply:
I know, I was wondering if there was any other particular reason for doing this specially in kafka tests? Maybe it was because of some issue when using kafka broker put in place by testcontainer? just curious.